spongehome.go

```diff
@@ -26,23 +26,65 @@
 package main
 
 import (
-    "github.com/SpongePowered/SpongeHome/controllers"
-    "github.com/go-macaron/pongo2"
-    "gopkg.in/macaron.v1"
+	"fmt"
+	"github.com/SpongePowered/SpongeHome/controllers"
+	"github.com/go-macaron/pongo2"
+	"github.com/sethvargo/go-fastly"
+	"gopkg.in/macaron.v1"
+	"os"
+	"time"
 )
 
+func clearfastly() {
+	//Clear Fastly cache
+	if os.Getenv("FASTLY_KEY") != "" {
+		time.Sleep(10 * time.Second) //let the http server startup
+		fmt.Println("starting fastly client")
+		client, err := fastly.NewClient(os.Getenv("FASTLY_KEY"))
+		if err != nil {
+			fmt.Println(err)
+		}
+		services, err := client.ListServices(&fastly.ListServicesInput{})
+		FASTLY_NAME := ""
```

Comment:
Please don't name this FASTLY_NAME, it's not a constant (nor should constants be named like that in Go)